command/genesis/genesis.go

```diff
@@ -185,6 +185,13 @@ func setFlags(cmd *cobra.Command) {
 			command.DefaultPremineBalance,
 			"the amount which will be premined to all the validators",
 		)
+		cmd.Flags().StringVar(
+			&params.integrate,
+			integrateFlag,
```

Comment:
`integrate` is not clear for me and is it only for Polybft right? How about `polybft-predeploy`?

Reply:
Agree, as we're just back-porting from v3, I went with it because I don't know the original motivation for this naming, @0xPolygon/ethernal can you give some context?

Reply:
It is the same flag, which we introduced in a v3-contracts [PR](https://github.com/0xPolygon/polygon-edge/pull/796/files#diff-c895b811654edb8f4e81a754e99853dd1a3ed355bee401d792c24d8e90d50274R189). :slightly_smiling_face: 
It represents root folder where smart contracts reside.

Reply:
Agree, we have called it "contracts-path". Think it is reasonable short and meaningful.

Reply:
ok, renamed, yeah we'll have conflict 😸